llvm_util/known_fns.cpp

```diff
@@ -516,8 +517,16 @@ known_call(llvm::CallInst &i, const llvm::TargetLibraryInfo &TLI,
     RETURN_EXACT();
 
   auto decl = i.getCalledFunction();
+  if (!decl)
+    RETURN_EXACT();
+
+  // Intel X86 intrinsics
+  if (decl->hasName() && decl->getName().starts_with("__fksv"))
```

Comment:
what's this? can you give a link to some documentation around __fksv_* functions?

If LLVM doesn't recognize them as library calls, this code is not correct in general.